js/client-chat.js

```diff
@@ -1237,6 +1237,13 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'notify':
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					this.notifyOnce(row[1], "\"" + row.slice(2).join('|') + "\"", 'highlight');
```

Comment:
Other notifications (read highlights) have "quotes" around the notification message, as seen here https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/js/client-chat.js#L1493